src/bin/pgcopydb/cli_list.c

```diff
@@ -221,6 +222,29 @@ cli_list_getenv(ListDBOptions *options)
 		++errors;
 	}
 
+	/* when --split-max-parts has not been used, check PGCOPYDB_SPLIT_MAX_PARTS */
```

Comment:
Is there a way to reuse `cli_copydb_getenv()` from here?

Reply:
`cli_copydb_getenv()` reads environment variables and fills in the `CopyDBOptions` values.

`cli_list_getenv()` does a similar thing but it fills in `ListDBOptions` instead.

We can consider creating a helper function that is called by both of these functions and fill in the values for the relevant structure. I can not think of a clean way to implement this as we do not have generic functions that enables writing reusable code that adapts to different data types.

We can consider doing this in a follow-up refactor PR.

Reply:
Good point, forgot about that. Thanks for the clear reminder.